aea/cli/config.py

```diff
@@ -39,11 +48,11 @@ def config(click_context):  # pylint: disable=unused-argument
 
 
 @config.command()
-@click.argument("JSON_PATH", required=True, type=AEAJsonPathType())
+@click.argument("JSON_PATH", required=True)  # , type=AEAJsonPathType())
```

Comment:
why commented out? remove if no longer needed

Reply:
not required anymore. removed